### PR TITLE
Rebase transmission to 3.16, migrate to s6v3

### PR DESCRIPTION
**Pull request by thespad**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-transmission/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

**Review by nemchik — APPROVED** [review]

File: root/etc/s6-overlay/s6-rc.d/init-transmission-config/run [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    sed -i "/\"peer-port\"/c\    \"peer-port\": ${PEERPORT}," /config/settings.json
+    sed -i '/peer-port-random-on-start/c\     "peer-port-random-on-start": false,' /config/settings.json
+fi
+
+# permissions
+chown abc:abc \
+    /config/settings.json \
+    /downloads \
+    /downloads/complete \
+    /downloads/incomplete \
```
Comment (leukanos) [inline root/etc/s6-overlay/s6-rc.d/init-transmission-config/run:48]: Why this change is required? For nfs mounted volumes it causes init issues on my end:
```
chown: changing ownership of '/downloads/complete': Operation not permitted
chown: changing ownership of '/downloads/incomplete': Operation not permitted
```

Comment (Roxedus) [reply root/etc/s6-overlay/s6-rc.d/init-transmission-config/run:48]: It is not a change, this file has only been moved. It would have had this error before this rebase too, but the default behaviour was to ignore errors.
